### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by addictif80** [jippi/docker-pixelfed#50]
Hello, i've followed this link for installing Pixelfed : https://jippi.github.io/docker-pixelfed/installation/guide/#configuration-quick-start
When i run setup script, i've an error message

```
Error: open .env: no such file or directory
Run 'dottie value --help' for usage.
```

But in my folder, .env file exist
```
drwxr-xr-x 9 root   root    4096 oct.   3 12:46 .
drwxr-xr-x 3 adrien adrien  4096 oct.   3 12:46 ..
-rw-r--r-- 1 root   root    5237 oct.   3 12:46 CODE_OF_CONDUCT.md
-rw-r--r-- 1 root   root    1096 oct.   3 12:46 docker-compose.migrate.yml
-rw-r--r-- 1 root   root   10293 oct.   3 12:46 docker-compose.yml
-rw-r--r-- 1 root   root     137 oct.   3 12:46 Dockerfile.docs
drwxr-xr-x 6 root   root    4096 oct.   3 12:46 docs
drwxr-xr-x 5 root   root    4096 oct.   3 12:46 docs-customization
-rw-r--r-- 1 root   root     705 oct.   3 12:46 .editorconfig
-rw-r--r-- 1 root   root   46020 oct.   3 12:46 .env
-rw-r--r-- 1 root   root   46020 oct.   3 12:46 .env.docker
drwxr-xr-x 8 root   root    4096 oct.   3 12:46 .git
-rw-r--r-- 1 root   root      12 oct.   3 12:46 .gitattributes
drwxr-xr-x 3 root   root    4096 oct.   3 12:46 .github
-rw-r--r-- 1 root   root     133 oct.   3 12:46 .gitignore
-rw-r--r-- 1 root   root     455 oct.   3 12:46 .hadolint.yaml
drwxr-xr-x 4 root   root    4096 oct.   3 12:46 images
-rw-r--r-- 1 root   root    1074 oct.   3 12:46 LICENSE
-rw-r--r-- 1 root   root     156 oct.   3 12:46 Makefile
-rw-r--r-- 1 root   root      91 oct.   3 12:46 .markdownlint.yml
-rw-r--r-- 1 root   root    3003 oct.   3 12:46 mkdocs.yml
-rw-r--r-- 1 root   root     183 oct.   3 12:46 package.json
-rw-r--r-- 1 root   root   26236 oct.   3 12:46 package-lock.json
drwxr-xr-x 3 root   root    4096 oct.   3 12:46 scripts
-rw-r--r-- 1 root   root     373 oct.   3 12:46 .shellcheckrc
drwxr-xr-x 2 root   root    4096 oct.   3 12:46 .vscode
```

Any idea ?
Best regards !

**Comment by annualstatement** [comment]
hey, i was also having trouble with the same error. I just figured it out that you need to install dottie from [https://github.com/jippi/dottie](url). Then you can rerun the script and should go through.

**Comment by addictif80** [comment]
Hello,
i've installed dottie on a fresh server for testing, and i've the same error. 
.env file does not exist.

**Comment by annualstatement** [comment]
Dang, I'm not sure how else to assist since it worked for me after that. However, it looks like your getting a different error than the original post. Sorry, I'm no expert at this.

**Comment by addictif80** [comment]
> Are you running the script from within the same directory?
Yes, i'm on the docker-pixelfed folder. In this folder, i can see scripts folder and .env file, between other folders/files.

**Comment by annualstatement** [comment]
> > Are you running the script from within the same directory?
> > Yes, i'm on the docker-pixelfed folder. In this folder, i can see scripts folder and .env file, between other folders/files.

Dang. Hopefully someone else can you help out.

**Comment by addictif80** [comment]
Sooo, i've tried again, and it work !
Thanks a lot

**Comment by annualstatement** [comment]
> Sooo, i've tried again, and it work ! Thanks a lot

Glad you figured it out.

**Comment by addictif80** [comment]
Yes, but i have a problem with mobile app, i can't upload anymore with Android device, i think this issue IS due to horizon, because i've tried to deploy other instance with ploi and it work. On my docker instance, horizon have no task failed/completed/running when i upload with official Android app or other pixelfed compatible apps.

**Comment by KamalF** [comment]
Hi, I encountered the same problem. Installing dottie fixed it indeed. Looking at the source I wonder if it is not an issue with the users and using dottie as a container.
I tried on today's patch (8f077b151ca52bc6646bed275b3079e34cfa87ae) on a Debian 12 with a user "debian".

**Comment by jippi** [comment]
I fixed a bunch of stuff in `main` (and releases) the last couple of weeks, including end-to-end testing in CI verifying that project can bootstrap correctly following the guide.

Please let me know if there are any remaining issues after testing the latest release (or main)
